src/lev-dashboard/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import LevPage from '../lev-page';
+import DashboardContent from "./DashboardContent";
+
+const LevDashboard = (props) =>
+<LevPage
+  title="LEV Dashboard"
+  topnav={{
+    department: 'HMPO',
+    serviceTitleText: 'Life Event Verification - Dashboard',
+    signOutLink: 'oauth/logout'
+  }}
+>
+  <script dangerouslySetInnerHTML={{ __html: `window.dashboardProps=${JSON.stringify(props)};` }}></script>
```

Comment:
What is the purpose of this line?

Reply:
`hydrate` is the great react con, it does nothing clever or magical, it just forces you to rebuild whatever components you wish to use on the frontend. To build a component, you need its `props` (in this case the data) it was built with on the server. Rather than make an ajax request back to the server - we inject the data we need into a script tag here.

Reply:
Okay. In that case, I think this should also go into the renderer as another argument.

Reply:
We've agreed this should live in the renderer eventually but it can live here for now.